reinforcement_learning/rl_bin_packing_newsvendor_vehicle_routing_ray_customEnv/rl_news_vendor_ray_custom.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Remove the discrete version. It doesn't work. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>